freeipa/src/main/java/com/sequenceiq/freeipa/repository/ChildEnvironmentRepository.java

```diff
@@ -15,9 +15,22 @@
 @Transactional(Transactional.TxType.REQUIRED)
 public interface ChildEnvironmentRepository extends CrudRepository<ChildEnvironment, Long> {
 
-    @Query("SELECT c.stack FROM ChildEnvironment c WHERE c.environmentCrn = :childEnvironmentCrn AND c.stack.accountId = :accountId")
+    @Query("SELECT c.stack FROM ChildEnvironment c WHERE c.environmentCrn = :childEnvironmentCrn AND c.stack.accountId = :accountId AND c.stack.terminated = -1")
```

Comment:
Why did this query change?

Reply:
With the rebuild, there can be multiple rows for a single environment. We want the row that hasn't been terminated yet.

Reply:
@doktoric This code change will affect existing code paths.